bayes_kit/rhat.py

```diff
@@ -1,11 +1,88 @@
 import numpy as np
 from numpy.typing import NDArray, ArrayLike
+import scipy as sp
 
 FloatType = np.float64
 VectorType = NDArray[FloatType]
 SeqType = ArrayLike
 
 
+def split_chains(chains: list[SeqType]) -> list[SeqType]:
+    """
+    Return a list of the input chains split in half.  The result will be
+    a list twice as long as the input.  For example,
+    ```
+    >>> split_chains([[1, 2, 3, 4], [5, 6, 7]])
+    [[1, 2], [3, 4], [5, 6], [7]]
+    ```
+
+    Parameters:
+    chains: List of univariate Markov chains.
```

Comment:
This is true for all the docstrings in this file, and is difficult to really enforce until we have something like Sphinx building the docs to let us know, but these docstrings are not properly formatted Google style. 

The correct version would look like
```suggestion
    Args:
        chains: List of univariate Markov chains.
```

Reply:
fixed.